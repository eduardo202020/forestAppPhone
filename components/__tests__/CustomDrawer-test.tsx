import React from 'react';
import renderer from 'react-test-renderer';
import CustomDrawerContent from '../CustomDrawer';
import { SafeAreaProvider } from 'react-native-safe-area-context';

describe('CustomDrawerContent', () => {
  it('renderiza correctamente el drawer', () => {
    const props = { navigation: { closeDrawer: jest.fn() } };
    const tree = renderer
      .create(
        <SafeAreaProvider>
          <CustomDrawerContent {...props} />
        </SafeAreaProvider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
