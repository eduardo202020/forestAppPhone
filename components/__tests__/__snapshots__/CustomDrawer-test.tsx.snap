// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomDrawerContent renderiza correctamente el drawer 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;
